tests/sles4sap/saptune/mr_test.pm

```diff
@@ -95,6 +95,160 @@ sub test_bsc1152598 {
     assert_script_run "saptune revert all";
 }
 
+sub test_delete_rename {
+    my ($self) = @_;
+
+    my $dir  = "Pattern/testpattern_saptune-delete+rename";
+    my $note = "2161991";
+
+    ### Deleting a shipped Note (without override/with override + not applied/applied)
+
```

Comment:
Nothing important but you sometimes add blank line and sometimes not, please do the same every time ;-)

Reply:
The cases where I don't use a blank line are mini-scenarios.